docs/Configuration.md

```diff
@@ -302,6 +302,66 @@ shims:
         legacy-library-a: src/library.a
         legacy-library-b: src/library.b
 ```
+## Custom Bundling
+
+Custom bundles can be defined in the `ui5.yaml`. It should be located in the `builder` configuration. With the property `bundles` a list of `bundleDefinitions` can be described.
+
+```yaml
+builder:
+  bundles:
+    - bundleDefinition:
+        name: "sap-ui-custom.js"
+        defaultFileTypes:
+          - ".js"
+        sections:
+          - mode: raw
+            filters:
+            - ui5loader-autoconfig.js
+            resolve: true
+            sort: true
+        bundleOptions:
+          optimize: true
+    - bundleDefinition:
+        name: "app.js"
+        defaultFileTypes:
+          - ".js"
+        sections:
+          - mode: preload
+            filters:
+              - some/app/Component.js
+            resolve: true
+            sort: true
+          - mode: provided
+            filters:
+            - ui5loader-autoconfig.js
+            resolve: true
+        bundleOptions:
+          optimize: true
+  ```
+
+### Properties
+
+**bundleDefinitions**
```

Comment:
bundleDefinition?
This list is part of the `bundles` property

Reply:
Thanks, it should be the `bundles` property